dev.yml

```diff
@@ -3,7 +3,7 @@ name: mruby-engine
 up:
   - ruby:
       package: shopify/shopify/shopify-ruby
-      version: 2.2.3p172-shopify
```

Comment:
Is this change okay? Wanted to keep consisting with the version we are testing against in CI